packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,36 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isIIFE(path) {
```

Comment:
Not sure this is a real thing, but presumably this would break?

```js
var foo = 1;
var bar = 2;

const baz = ((foo) => {
  if (true) { return foo; } else { return 3; }
})(bar);
```

Your transform will make `baz` `1` instead of `2`. I think it might be worth just bailing out if there are any arguments.